src/arrays/referencing_array_and_its_elements/hard/q10/MinFinder.java

```diff
@@ -0,0 +1,36 @@
+package arrays.referencing_array_and_its_elements.hard.q10;
+
+/*
+Write a Java program to find the smallest number in a double array. The program should prompt the user to enter an array
```

Comment:
number of inputs needs to be higher and question changed so that the student can't just code a for loop where they take input and store it in only one variable depending if it is lower than the current value.

Reply:
We are using a method here that takes a double array so they can't store it in only one variable.